README.md

```diff
@@ -19,19 +20,35 @@ The server exposes APIs that are consumed by the web app and the mobile app.
    ```
    SECRET = secure_server_secret
    MONGODB_URI = mongodb://localhost:27017/boldo
-
+   
    # ###################### Online ######################
    # CLIENT_ADDRESS = https://boldo.penguin.software
    # KEYCLOAK_ADDRESS = https://sso-test.pti.org.py/auth
    IHUB_ADDRESS = https://sso-test.pti.org.py/api
-
    # ###################### Local ######################
    CLIENT_ADDRESS = http://localhost:3000
    KEYCLOAK_ADDRESS = http://localhost:8080/auth
+   BOLDO_CORS = http://localhost:3000
```

Comment:
is this the same as CLIENT_ADDRESS?
Also the above equivalent to this env variable should be described